MonoGame.Framework/Windows/WinFormsGamePlatform.cs

```diff
@@ -134,7 +134,7 @@ public override void StartRunLoop()
         
         public override void Exit()
         {
-            Application.Exit();
+            _window.Dispose();
```

Comment:
@christallire - What is this change for?  If you have a single window game will this still close the application correctly?